recipes/libm2k/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "libm2k" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/analogdevicesinc/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 64f62cf9a72d625c4a4344a1984a2f8c5d4a95aeaed0026a0cba7ac0c0ca7390
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libm2k', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - {{ cdt('libudev-devel') }}  # [linux]
+    - make  # [not win]
+    - ninja  # [win]
+  host:
+    - libiio
+    - libusb >=1
+    - python
+    - swig
+    - setuptools
+    - icu  # [osx]
+    - libiconv  # [osx]
+    - xz  # [osx]
+    - zlib  # [osx]
+  run:
+    - libiio
+    - python
+  run_constrained:
+    # require the MACOSX_DEPLOYMENT_TARGET set in conda_build_config.yaml
+    # (10.11 to be able to compile bonjour support)
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+
+test:
+  commands:
+    # verify that executables can be run
+    - m2kcli --help
+
+    # verify that headers get installed
+    - test -f $PREFIX/include/libm2k/m2k.hpp  # [not win]
+    - if not exist %PREFIX%\\Library\\include\\libm2k\\m2k.hpp exit 1  # [win]
+
+    # verify that libraries get installed
+    - test -f $PREFIX/lib/libm2k${SHLIB_EXT}  # [not win]
+    - if not exist %PREFIX%\\Library\\bin\\libm2k.dll exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\libm2k.lib exit 1  # [win]
+  imports:
+    - libm2k
+
+about:
+  home: https://github.com/analogdevicesinc/libsmu
```

Comment:
Should this and below refer to `libm2k` instead of `libsmu`?

Reply:
Fixed.
libsmu will be submitted after this one. I think they are melding together in my mind.